projects/admin/src/app/record/brief-view/documents-brief-view/documents-brief-view.component.html

```diff
@@ -43,18 +43,18 @@ <h5 class="mb-0 card-title">
           {{ last && record.metadata.contribution.length > 3 ? '; …' : '' }}
         </li>
       </ul>
-      <!-- edition statement -->
-      <ng-container *ngIf="record.metadata.editionStatement">
-        <ul class="list-unstyled mb-0">
-          <li *ngFor="let edition of record.metadata.editionStatement">{{ edition._text[0].value }}</li>
-        </ul>
-      </ng-container>
       <!-- provision activity publications -->
       <ul class="list-unstyled" *ngIf="provisionActivityPublications.length > 0">
         <li *ngFor="let publication of provisionActivityPublications">{{ publication.value }}</li>
       </ul>
       <!-- shared part -->
       <shared-part-of [record]="record"></shared-part-of>
+      <!-- edition statement -->
+      <ng-container *ngIf="record.metadata.editionStatement">
```

Comment:
ng-container seems useless. *ngIf can be on `ul`, no?

Reply:
yes ! 👍